.github/ISSUE_TEMPLATE/lit-html-next-bug-report.md

```diff
@@ -0,0 +1,47 @@
+---
+name: lit-html lit-next preview bug report
+about: Create a report to help us improve
+title: '[lit-html] '
+labels: 'lit-next'
+assignees: ''
+
+---
+
+<!--
+Please read our contribution guidelines:
+  https://github.com/Polymer/lit-html/blob/master/CONTRIBUTING.md#filing-issues
+-->
+
+# Description
+A clear and concise description of what the bug is.
+
+# Steps to Reproduce
+
+1. Write this code
+
+```ts
+import {html, render} from 'lit-html';
+
+render(html`Reproduction Here`, document.body);
+```
+
+2. See this output...
+
+## Live Reproduction Link
+<!-- Please create a live reproduction on StackBlitz by forking this project: -->
+https://stackblitz.com/edit/lit-html-next
+
+## Expected Results
+<!-- A clear and concise description of what you expected to happen. -->
+
+## Actual Results
+<!-- Example: Error is thrown -->
+
+## Browsers Affected
+<!-- Check all that apply -->
+- [ ] Chrome
+- [ ] Firefox
+- [ ] Edge
+- [ ] Safari 11
```

Comment:
This should just be `Safari` now, and let's remove the rest until we support them